TicketBooking.Tests/Flights/Validators/FlightValidatorTests.cs

```diff
@@ -0,0 +1,218 @@
+﻿using AutoFixture;
+using FluentAssertions;
+using Moq;
+using TicketBooking.Flights.Models;
+using TicketBooking.Flights.Services;
+using TicketBooking.FileProcessor.Deserializer.Flight;
+using TicketBooking.Airports;
+using TicketBooking.Countries;
+using TicketBooking.Tests.Flights.Validators.TestData;
+
+namespace TicketBooking.Tests.Flights.Validators;
+
+public class FlightValidatorTests
+{
+    private readonly Fixture _fixture;
+    private FlightValidator _validator;
+    private readonly Mock<List<Flight>> _flightsMock;
+
+    public FlightValidatorTests()
+    {
+        _flightsMock = new();
+        _validator = new(_flightsMock.Object);
+        _fixture = new Fixture();
+    }
+
+    [Theory]
+    [InlineData(1, true)]
+    [InlineData(3089799, true)]
+    [InlineData(4234, true)]
+    public void ValidateId_ReturnTrue(int id, bool expectedResult)
+    {
+        // Arrange
+        // Done
+
+        // Act
+        bool idValidation = _validator.validateId(id);
+
+        // Assert
+        idValidation.Should().Be(expectedResult);
+    }
+
+    [Theory]
+    [InlineData(-5, false)]
+    [InlineData(-43248, false)]
+    [InlineData(-747382, false)]
```

Comment:
since we know that it returns false, no need to read `expectedResult` from the InlineData